src/category_theory/concrete_category/bundled_hom.lean

```diff
@@ -85,6 +85,44 @@ has_forget₂.mk'
   @map
   (by intros; apply heq_of_eq; apply h_map)
 
+variables {d : Type u → Type u}
+variables (hom)
+
+/--
+The `hom` corresponding to first forgetting along `F`, then taking the `hom` associated to `c`.
```

Comment:
Please add something like "This is useful to construct bundled homs in `CommMon` from bundled homs in `Mon`."

Reply:
Done.